pkg/lib/bundle/testdata/validate/valid_bundle/metadata/dependencies.yaml

```diff
@@ -7,3 +7,5 @@ dependencies:
       group: test.coreos.com
       kind: testapi
       version: v1
+  - type: olm.label
```

Comment:
Is the olm.package dependency in this fixture wrong? That's nice ammunition for my complaining about unmaintainable test fixtures.

Reply:
The ones that are wrong here are the gvk and label ones. They need to under `value:` like package. This is my fault. Also this bundle is used for manifests validation AKA CRD and CSV, not metadata so I remove this file altogether to avoid confusion.